src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('tbody');
+const rowsArray = Array.from(document.querySelectorAll('tbody tr'));
+const headings = document.querySelector('thead');
+
+function convertToNumber(string) {
+  return parseInt(string.slice(1).replaceAll(',', ''));
+}
+
+headings.addEventListener('click', e => {
+  const index = e.target.cellIndex;
+
+  rowsArray.sort((a, b) => {
+    const aText = a.cells[index].textContent;
+    const bText = b.cells[index].textContent;
+    const callWithA = convertToNumber(aText);
+    const callWithB = convertToNumber(bText);
+
+    if (+aText) {
+      return +aText - (+bText);
+    } else if (+callWithA) {
```

Comment:
You've converted it to a number already 
```suggestion
    } else if (callWithA) {
```